core/lowering/passes/remove_dropout.cpp

```diff
@@ -8,85 +9,7 @@ namespace lowering {
-            %6 = aten::dropout(%input, %4, %5)
-            return (%6))IR";
-  std::string no_dropout_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
-
-  torch::jit::SubgraphRewriter remove_dropout;
-  remove_dropout.RegisterRewritePattern(dropout_pattern, no_dropout_pattern);
-  remove_dropout.runOnGraph(graph);
-
-  std::string dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            %6 = aten::dropout_(%input, %4, %5)
-            return (%6))IR";
-  std::string no_dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
-
-  torch::jit::SubgraphRewriter remove_dropout_inplace_pattern;
-  remove_dropout_inplace_pattern.RegisterRewritePattern(dropout_inplace_pattern, no_dropout_inplace_pattern);
-  remove_dropout_inplace_pattern.runOnGraph(graph);
-
-  // remove feature_dropout
-  std::string feature_dropout_pattern = R"IR(
-        graph(%input, %4, %5):
-            %6 = aten::feature_dropout(%input, %4, %5)
-            return (%6))IR";
-  std::string no_feature_dropout_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
-
-  torch::jit::SubgraphRewriter remove_feature_dropout_pattern;
-  remove_feature_dropout_pattern.RegisterRewritePattern(feature_dropout_pattern, no_feature_dropout_pattern);
-  remove_feature_dropout_pattern.runOnGraph(graph);
-
-  // remove feature_dropout inplace
-  std::string feature_dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            %6 = aten::feature_dropout_(%input, %4, %5)
-            return (%6))IR";
-  std::string no_feature_dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
-
-  torch::jit::SubgraphRewriter remove_feature_dropout_inplace_pattern;
-  remove_feature_dropout_inplace_pattern.RegisterRewritePattern(
-      feature_dropout_inplace_pattern, no_feature_dropout_inplace_pattern);
-  remove_feature_dropout_inplace_pattern.runOnGraph(graph);
-
-  // remove feature_alpha_dropout
-  std::string feature_alpha_dropout_pattern = R"IR(
-        graph(%input, %4, %5):
-            %6 = aten::feature_alpha_dropout(%input, %4, %5)
-            return (%6))IR";
-  std::string no_feature_alpha_dropout_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
-
-  torch::jit::SubgraphRewriter remove_feature_alpha_dropout_pattern;
-  remove_feature_alpha_dropout_pattern.RegisterRewritePattern(
-      feature_alpha_dropout_pattern, no_feature_alpha_dropout_pattern);
-  remove_feature_alpha_dropout_pattern.runOnGraph(graph);
-
-  // remove feature_alpha_dropout inplace
-  std::string feature_alpha_dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            %6 = aten::feature_alpha_dropout_(%input, %4, %5)
-            return (%6))IR";
-  std::string no_feature_alpha_dropout_inplace_pattern = R"IR(
-        graph(%input, %4, %5):
-            return (%input))IR";
```

Comment:
The new lowering pass **does not** remove these two operators, currently. See the discussion in the main description of the PR for additional details.